OrbitGl/CaptureSerializer.cpp

```diff
@@ -167,8 +155,8 @@ void CaptureSerializer::Load(const std::string& filename) {
     }
 
     GOrbitApp->FireRefreshCallbacks();
+    GOrbitApp->AddSamplingReport(Capture::GSamplingProfiler, GOrbitApp.get());
```

Comment:
The reason this was afterwards is that I had a problem with some `DataView::OnDataChanged` having to be called first and getting a crash otherwise. But I don't remember the details and I really can't reproduce it now, it might have been fixed by a following commit, so I moved it before `FireRefreshCallbacks`.